util/common.py

```diff
@@ -28,6 +28,32 @@
 ANNOTATION_FILE_FORMAT = '%s_annotations_%s.json'
 
 
+class QartodFlags:
+    """Primary flags for QARTOD."""
+    # Don't subclass Enum since values don't fit nicely into a numpy array.
+    PASS = 1
+    NOT_EVALUATED = 2
+    SUSPECT = 3
+    FAIL = 4
+    MISSING = 9
+
+    @classmethod
+    def getFlagOrder(cls):
+        return [cls.NOT_EVALUATED, cls.PASS, cls.MISSING, cls.SUSPECT, cls.FAIL]
+
+    @classmethod
+    def getValidQCFlags(cls):
+        return [cls.PASS, cls.NOT_EVALUATED, cls.SUSPECT, cls.FAIL, cls.MISSING]
+
+    @classmethod
+    def getQCFlagMeanings(cls):
+        return ["PASS", "NOT_EVALUATED", "SUSPECT", "FAIL", "MISSING"]
+
+    @classmethod
+    def isValidQCFlag(cls, flag):
+        return flag in QartodFlags.getValidQCFlags()
```

Comment:
Could you also state "in cls.getValidQCFlags()"? Not suggesting you change to that, just asking.